task/sast-snyk-check/0.1/README.md

```diff
@@ -0,0 +1,22 @@
+# sast-snyk-check task
+
+## Description:
+
+sast-snyk-check task uses [Snyk Code](https://snyk.io/product/snyk-code/) tool to perform Static Application Security Testing (SAST) for [Snyk](https://snyk.io/), a popular cloud-native application security platform.
+
+Snyk's SAST tool uses a combination of static analysis and machine learning techniques to scan an application's source code for potential security vulnerabilities, including common issues such as SQL injection, cross-site scripting (XSS), and code injection attacks.
+
+This task is executed only if the user provides a Snyk token stored in a secret in their namespace. The name of the secret then needs to be supplied in the `snyk-secret` pipeline parameter.
+
+## Params:
+
+| name        | description                                         |
+|-------------|-----------------------------------------------------|
+| SNYK_SECRET | Name of the secret which contains the snyk_token.    |
+| ARGS        | Extra args needs to append.                         |
```

Comment:
```suggestion
| ARGS        | Append arguments.                         |
```
Refer to https://github.com/redhat-appstudio/build-definitions/pull/331/files#diff-800129ea369b045abdc7b5f644036246478c619ad421550f08761ecb69ab694eR22

Reply:
updated, thanks!